static/js/components/util/panel.vue

```diff
@@ -8,18 +8,18 @@
               <b-form-row>
                 <b-col class="text-left">
                   <i 
-                    class="align-middle mx-2" 
+                    class="align-middle fa-lg mx-2" 
```

Comment:
the fa-lg works without having a font awesome icon?

Reply:
Oh, this does actually get a font awesome icon. On the next line there is `:class="icon"` - this is binding the icon class that is passed into the component as a prop.

Reply:
ahhh